src/debugger/appWorker.ts

```diff
@@ -218,6 +224,13 @@ postMessage({workerLoaded:true});`;
         if (!this.sourcesStoragePath)
             throw ErrorHelper.getInternalError(InternalErrorCode.SourcesStoragePathIsNullOrEmpty);
         this.webSocketConstructor = webSocketConstructor;
+        this.DOM_LOCATION_STUB = `// https://github.com/callstack/haul/pull/641/files#diff-7819345467dc6ec57baa550e3f60938cR123
+// Haul packager since 0.13.0 relies on DOM location object presence in global object
+// but since worker is ran as a Node process it doesn't have a location, so we add it as packager URL address
+if (!self.location) {
+    self.location = new URL('http://${this.packagerAddress}:${this.packagerPort}');
+}
+`;
```

Comment:
I can hide `self.location.host` assignment behind an `if` block here https://github.com/callstack/haul/pull/641/files#diff-7819345467dc6ec57baa550e3f60938cR123 in Haul, so you wouldn't need that stub.

Reply:
Great, thanks. Then I'll rollback that change.

Reply:
@zamotany When this change could be released?

Reply:
@SounD120 I'll try to do it today.